source/includes/starting-compass-paste-string.rst

```diff
@@ -28,7 +27,20 @@
       .. include:: /includes/fact-hide-connection-string-password.rst
             
       To learn how to format the connection string for a deployment that is not 
-      hosted on |service|, see :manual:`Connection String URI Format </reference/connection-string>`.
+      hosted on |service|, see :manual:`Connection String URI Format
+      </reference/connection-string>`.
+      
+   .. step:: Name your connection (Optional).
+
+      Use the name field to enter a name for your connection. If you do
+      not specify a name, Compass uses the cluster's hostname as the the
+      connection name. 
+
+   .. step:: Choose a color for your connection (Optional).
+
+      Use the color drop down menu to select a label color for your
```

Comment:
[nit] [style guide - terminology](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#:~:text=contains%20your%20file.-,drop%2Ddown,-Use%20as%20an) drop-down menu is hyphenated!
```suggestion
      Use the color drop-down menu to select a label color for your
```

Reply:
Fixed!